roles/osasinfra/templates/clientpod.yml.j2

```diff
@@ -0,0 +1,103 @@
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  name: {{ cifmw_osasinfra_client_pod_name }}
+  namespace: openstack
+spec:
+  containers:
+  - args:
+    - infinity
+    command:
+    - /bin/sleep
+    env:
+    - name: CONFIG_HASH
+      value: n6h58h584h689h588hfdh687h8dh64dh685h5c5h5dfh578h648h5c6h576h54fh559h65bh5bh674h59bh678h655h5ddh55hb7h7hb4h5bch69h668q
+    - name: OS_CLOUD
+      value: default
+    image: {{ cifmw_osasinfra_client_pod_image }}
+    imagePullPolicy: Always
+    name: {{ cifmw_osasinfra_client_pod_name }}
+    resources: {}
+    securityContext:
+      privileged: true
+    terminationMessagePath: /dev/termination-log
+    terminationMessagePolicy: File
+    volumeMounts:
+    - mountPath: /home/cloud-admin/.original-config/openstack/clouds.yaml
+      name: openstack-config
+      subPath: clouds.yaml
+    - mountPath: /home/cloud-admin/.original-config/openstack/secure.yaml
+      name: openstack-config-secret
+      subPath: secure.yaml
+    - mountPath: /etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem
+      name: combined-ca-bundle
+      readOnly: true
+      subPath: tls-ca-bundle.pem
+    - mountPath: /var/run/secrets/kubernetes.io/serviceaccount
```

Comment:
This is problematic too. We should remove it because it's loading the underlying OCP serviceaccount